lib/GoogleDocsCodePaste/rtf_template.lua

```diff
@@ -62,7 +62,7 @@ local function generateHighlightedCode(code, language)
   io.write(code)
   io.close(file)
 
-  local codeRtf = hs.execute("cat /tmp/code.txt | /usr/local/bin/highlight --no-trailing-nl -O rtf --font 'Roboto Mono' --font-size 16 --syntax " .. language .. " -s 'solarized-light'")
+  local codeRtf = hs.execute("cat /tmp/code.txt | highlight --no-trailing-nl -O rtf --font 'Roboto Mono' --font-size 16 --syntax " .. language .. " -s 'solarized-light'", true)
```

Comment:
@streeter I knew this day would come… the problem is that the Hammerspoon exec is not very `$PATH` aware in my experience, and it really prefers full paths. We could run it through `bash -c "..."` and try to get a shell going with PATH, but also the user might use ZSH/Fish/etc.

Since it's one binary path, it might be just worth checking both locations for an executable file, and take whichever one we find first.